accommodations/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+
+from users.models import Base, User
+
+class Accommodation(Base):
+    name           = models.CharField(max_length=250)
+    description    = models.CharField(max_length=250)
+    price          = models.DecimalField(decimal_places=2, max_digits=4)
+    address        = models.CharField(max_length=250)
+    region         = models.CharField(max_length=30)
+    is_verified    = models.BooleanField(default=False)
+    latitude       = models.CharField(max_length=100)
+    longtitude     = models.CharField(max_length=100)
+    check_in_time  = models.CharField(max_length=30)
+    check_out_time = models.CharField(max_length=30)
```

Comment:
`2022-01-02 11:00:00`까지 저정해야 하므로 `datetime` 필드를 사용하는게 좋겠습니다.
```suggestion
    check_in_time  = models.
    check_out_time = models.
```

Reply:
병민님 안녕하세요! 이 부분은 숙소 정보에 포함되는 체크인/체크아웃 시간인데요.
예를 들면, 호스트가 체크인 시간: 14:00 PM, 체크아웃 시간: 10:00 AM 이렇게 들어가서 CharField로 처리하려도 했습니다. 어떻게 생각하시나요?

Reply:
`datetime` 필드를 이용해도 설명해주신 부분과 동일하게 string으로 리턴해줄 수 있습니다. 굳이 다른 의미를 가진 필드로 저장할 필요가 없어보이네요~

Reply:
네 알겠습니다. 반영해서 수정하도록 하겠습니다! 감사합니다 😃